ConsumerApi.Tests.Integration/Models/Token.cs

```diff
@@ -13,4 +13,24 @@ public class Token
     public string? CreatedBy { get; set; }
     public string? CreatedByDevice { get; set; }
     public string? Content { get; set; }
+
+    public bool ShouldSerializeExpiresAt()
+    {
+        return ExpiresAt != null;
+    }
+
+    public bool ShouldSerializeCreatedBy()
+    {
+        return CreatedBy != null;
+    }
+
+    public bool ShouldSerializeCreatedByDevice()
+    {
+        return CreatedByDevice != null;
+    }
+
+    public bool ShouldSerializeContent()
+    {
+        return Content != null;
+    }
```

Comment:
These aren't used anywhere. And I don't understand what they should be used for anyway.

Reply:
These are used behind the scenes for the Schema validations, they are a pattern of Newtonsoft.JSON.
This way, I can use the Token class to validate both POST and GET responses, as opposed to having the CreateTokenResponse that we had before.